server/test/api/vouchers.js

```diff
@@ -0,0 +1,152 @@
+/* jshint expr: true */
+/* global describe, it, beforeEach */
+
+var chai = require('chai');
+var expect = chai.expect;
+
+/** import test helpers */
+var helpers = require('./helpers');
+var uuid    = require('node-uuid');
+helpers.configure(chai);
+
+
+/**
+* The /vouchers API endpoint
+*
+* @desc This test suit is about the vouchers transactions
+*
+*/
+
+describe('The /vouchers HTTP endpoint ::', function () {
+  'use strict';
+
+  var agent = chai.request.agent(helpers.baseUrl);
+
+  /** login before each request */
+  beforeEach(helpers.login(agent));
+
+  /** Test with dates */
+  var date = new Date();
+
+  var voucher = {
+    uuid : uuid.v4(),
+    date : date,
+    project_id : 1,
+    reference : 'TPA1',
+    currency_id : 1,
+    amount : 10,
+    description : 'Voucher transaction',
+    document_uuid : uuid.v4(),
+    user_id : 1
+  };
+
+  var voucher_item = {
```

Comment:
`voucher_item` should be an array of items to have a balanced transaction.  Can you make an array of two or three items to make sure it works?